### PR TITLE
Topics

### PR DESCRIPTION
**Pull request by TeriPastorino**
@levicole 
and up top it says can't merge, but that is ok probably

### CODE REVIEW

File: app/views/bookmarks/show.html.erb [hunk 1]
```diff
@@ -1,2 +1,29 @@
...
+</div>
+
+div class="row">
+  <div class="col-md-8">
+    <% @bookmarks.each do |bookmark| %>
+      <div class="media">
+        <h4 class="media-heading">
+          <%= link_to [@topic, bookmark.url] %>
+          <div class"col-md-8">
+            <%= link_to "Delete Bookmark", [@topic, bookmark.url], method: :delete, class: 'btn btn-danger' %>
```
Comment (levicole) [inline app/views/bookmarks/show.html.erb]: don't use bookmark.url here.

bookmark.url is the path to the external url. We need to use just the bookmark object to create an internal URL for us to delete.

File: app/views/bookmarks/show.html.erb [hunk 2]
```diff
@@ -1,2 +1,29 @@
...
+    <% end %>
+  </div>
+</div>
+
+div class="row">
+  <div class="col-md-8">
+    <% @bookmarks.each do |bookmark| %>
+      <div class="media">
+        <h4 class="media-heading">
+          <%= link_to [@topic, bookmark.url] %>
```
Comment (levicole) [inline app/views/bookmarks/show.html.erb]: Do we want to link to the external URL here? If so, we only need bookmark.url, not [@topic, bookmark.url]

File: app/views/topics/index.html.erb [hunk 3]
```diff
@@ -1,9 +1,15 @@
+<p>Find me in app/views/topics/index.html.erb</p>
```
Comment (levicole) [inline app/views/topics/index.html.erb]: Let's remove this.

File: app/views/topics/index.html.erb [hunk 4]
```diff
@@ -11,34 +17,18 @@
       <h4 class="media-heading">
         <%= link_to topic.title, topic %>
       </h4>
+      <small>
+          <% if policy(topic).destroy? %>
+          <%= link_to "Delete Topic", [@topic, topic], method: :delete, class: 'btn btn-default btn-sm', data: { confirm: 'Are you sure you want to delete this topic?' } %>
```
Comment (levicole) [inline app/views/topics/index.html.erb]: You only need topic here, not [@topic, topic].

File: app/views/topics/show.html.erb [hunk 5]
```diff
@@ -1,24 +1,17 @@
-<h1>@topic.name</h1>
+<h1><%= @topic.title %></h1>
 <p>Find me in app/views/topics/show.html.erb</p>
```
Comment (levicole) [inline app/views/topics/show.html.erb:1]: let's remove this.

File: app/views/topics/index.html.erb (hunk 3 above)
Comment (TeriPastorino) [inline app/views/topics/index.html.erb]: removed, it was sort of helping me keep track of where i was

File: app/views/topics/show.html.erb (hunk 5 above)
Comment (TeriPastorino) [inline app/views/topics/show.html.erb:1]: done, thanks
cept i think i was putting this here to show the topic selected and then the links would display underneath on the page? but i could be in the wrong place?

File: app/views/bookmarks/show.html.erb (hunk 1 above)
Comment (TeriPastorino) [inline app/views/bookmarks/show.html.erb]: on line 20 or 22 or both? sorry i was viewing it differently - trying to figure out the best way to display these to see them properly

**Comment by TeriPastorino** [comment]
added some comments and a question

**Pull request merged**